packages/rpc/src/__tests__/client.test.ts

```
@@ -0,0 +1,472 @@
+			const createProxy = (
+				baseClient: any,
+				baseUrl: string,
+				path: string[] = [],
+			) => {
+				return new Proxy(baseClient, {
+					get(target, prop, receiver) {
+						if (typeof prop === "string") {
+							const routePath = [...path, prop];
+
+							if (prop === "$get") {
+								return async (...args: any[]) => {
```

Biome (1.9.4)

[error] This variable is unused.

Unused variables usually are result of incomplete refactoring, typos and other source of bugs.
Unsafe fix: If this is intentional, prepend args with an underscore.


(lint/correctness/noUnusedVariables)